### ISSUE TITLE
Homepage animation: SVG mask issues in Chrome 60

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#423]
Worked fine in Chrome ≤ 59, but as of 60 the homepage animation looks like this at most sizes:

![svg-mask-bug](https://user-images.githubusercontent.com/69633/28730130-b3dcf06c-7383-11e7-8b54-87d2fbdf015e.gif)

Already attempted:

- Using `clip-path` instead of `mask`. This would require a serious refactor because this is currently a multi-shape mask. It would also require the introduction of a new dependency to manage shape morphing. Alternatively, we could remove the audio/headphones state of the animation, which would allow us to consolidate to a single shape.
- Using pixel instead of percentage values for the rectangle that appears to be bleeding through. Unfortunately, this doesn't seem to make a difference.
- [Embedding the animation inline](https://codepen.io/tylersticka/pen/d85119d8bae5876a94a7a6a5cb090a02/). No change.
- Giving the element a background color (not transparent). No change.
- Updating `TweenMax`. No change.

After confirming that the animation still worked in other browsers, I submitted a Chromium issue: https://bugs.chromium.org/p/chromium/issues/detail?id=750252

In the meantime, there is an Opera Mini check we could extend to also check for Chrome 60 (or later, to be safe) to disable the animation.

**Comment by tylersticka** [comment]
Oh good, it actually happens even with the fallback:

<img width="754" alt="screen shot 2017-07-28 at 11 26 20 am" src="https://user-images.githubusercontent.com/69633/28731260-19d47788-7388-11e7-8307-a6c1fb991143.png">

Commented on my original ticket: https://bugs.chromium.org/p/chromium/issues/detail?id=750252#c1

**Comment by tylersticka** [comment]
Quick-fix was deployed today, updated the Chromium ticket to let them know that example is no longer valid: https://bugs.chromium.org/p/chromium/issues/detail?id=750252

Going to create a new issue to track re-implementing the animation once that issue is merged and available to end users (likely next month).
